src/client/opamClient.ml

```diff
@@ -1602,9 +1602,10 @@ let update_with_init_config ?(overwrite=false) config init_config =
 
 let check_for_sys_packages config system_packages =
   if system_packages <> [] then
-    let ((missing, _) as set) =
+    let (available, required, missing) =
```

Comment:
fun fact: the original code got it backwards. I'm not sure what it means in terms of bugs though. We might want to have a new test case for this.

Reply:
Oh dear 😅 Thanks for fixing this Kate!